test/test_cuda.py

```diff
@@ -3642,6 +3643,7 @@ def alloc(n):
         del a, b, c
         # force release_cached_blocks to run with some expandable segments in the free list
         alloc(120)
+        torch.cuda.memory.set_per_process_memory_fraction(1.0)
```

Comment:
What if the test fails? Then the memory fraction will never be set and subsequent tests will fail too?